vision/image_simulator/cnn_letters.py

```diff
@@ -0,0 +1,129 @@
+from __future__ import absolute_import
```

Comment:
Need some more comments in this file about what's going on (can be high level stuff)